docs/ManageTasks.rst

```diff
@@ -172,31 +117,143 @@ A project is defined by these properties:
-By default, project's Tasks are displayed on the BOARD view.
+To access the list of a project's tasks, you just need to click on it's title on the CARD. You can choose between the two views BOARD or LIST.
+By default, project's tasks are displayed on the BOARD view.
 
 
 |NewTask09|
 
+
+.. _AddTask:
+
+How to add a new task?
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can add a task either in your Tasks list i.e. under TASKS tab or under a project in which you are at least participant. 
+For that you just need to click on "Add Task" button |NewTask07| on the top left of the page.
+A drawer opens allowing you to fill in the tasks' fields. 
+
+|NewTask08|
+
+.. tip:: The title field is mandatory to enable tasks's creation.
+
+.. note:: The differences between a task added under TASKS tab and under a project:
+          
+          -  In TASKS tab, only the assignee filed is pre-filled, and by default assigned to you.
+          
+          -  Under a project, only the project field is pre-filled, by default set to the project under which you are adding a new task.
+
+
 You can edit a task by clicking on its title which opens the "Edit task" drawer.
 
 |NewTask10|
 
+.. _SearchTask:
+
+How to search for a task?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Searching for a task is made the same in both tabs: TASKS and under a project.
+
+To search for a task you can either use the filter field allowing to search a task through a keyword or use the filter drawer which allows to make 
+an advanced search with multiple criteria. It also allows to group and sort tasks.
+
+.. _SearchTaskFilterField:
+
+Search for a task using the Filter field
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To use the first alternative .i.e. the filter field, you just need to type a keyword in the field. The returned results correspond to tasks containing the keyword in the 
+title or the description.
+
+|NewTask06|
+
+
+.. _SearchTaskFilterBtn:
+
+Search for a task using the Filter button
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The filter button allows you to make an advanced search and/or sort by setting the desired choices in 3 differents tabs.
+
+Group and Sort tab
+````````````````````
+The group and sort tab allows you to group and/or sort tasks according to the selected choices.
+
+|NewTask15|
+
+The "Group by" options organizes the tasks and groups them by a common criteria: Those belonging to the same project together, 
+those having the same due date (overdue, incoming...) together, etc...
+
+The "Sort by" options allows to sort tasks in an order according to a criteria. For example when sorting by priority, tasks with the highest priority appear first.
+
+.. note:: Note that some critera in the "Group and Sort" tab differs between project's tasks and TASKS. FOr example, grouping project's tasks by project does not have sens
```

Comment:
```suggestion
.. note:: Note that some criteria in the "Group and Sort" tab differ between project's tasks and TASKS  tab. For example, grouping the tasks of a project by project does not make sense, so that option is not offered.
```